### PR TITLE
contentplayback is contentplay-back

### PR DESCRIPTION
**Pull request by incompl**

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -464,6 +464,16 @@ const contribAdsPlugin = function(options) {
           window.clearTimeout(player.ads.cancelPlayTimeout);
           player.ads.cancelPlayTimeout = null;
         }
+
+        // This was removed because now that "playing" is fixed to only play after
+        // preroll, any integration should just use the "playing" event. However,
+        // we found out some 3rd party code relied on this event, so we've added
+        // it back in to give people more time to update their code.
```
Comment (ldayananda) [inline src/plugin.js]: Perhaps add `temporarily` to make it obvious this is going away at some point?

**Comment by ldayananda** [comment]
Last thing would be to update the 4.0 migration guide and documentation related to this to indicate deprecation rather than removal.

Otherwise, LGTM

**Comment by incompl** [comment]
Technically it was removed in 4.0.0 and will be re-added in a later version
so I want to leave it in there. It feels dishonest to say we deprecated it
after the fact.

**Comment by ldayananda** [comment]
True, perhaps it will cause less confusion if we specify the exact versions in which it was removed and re-added. That is, unless this change is going to be a major update.

I still feel the README should be updated to reflect this change.

**Comment by incompl** [comment]
I'll add it to the readme as a warning to not use it

**Comment by bcvio** [comment]
LGTM

**Comment by pagarwal123** [comment]
QA: LGTM

**Comment by gkatsev** [comment]
LGTM

**Pull request merged**